apps/api/src/app/subscribers/e2e/update-preference.e2e.ts

```diff
@@ -0,0 +1,55 @@
+import { UserSession } from '@novu/testing';
+import { expect } from 'chai';
+import axios from 'axios';
+import { NotificationTemplateEntity } from '@novu/dal';
+import { ChannelTypeEnum, IUpdateSubscriberPreferenceDto } from '@novu/shared';
+import { getPreference } from './get-preferences.e2e';
+
+const axiosInstance = axios.create();
+
+describe('Update Subscribers preferences - /subscribers/:subscriberId/preference/:templateId (PATCH)', function () {
```

Comment:
```suggestion
describe('Update Subscribers preferences - /subscribers/:subscriberId/:templateId/preference (PATCH)', function () {
```

Reply:
Hey, @scopsy are you sure?
Looks like the route is `@Patch('/:subscriberId/preference/:templateId')` like you suggested above. or am i missing something. :)